### PR TITLE
Performance improvements on element creation

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

I found this while investigating some performance issues: when we fetch a large amount of elements we run some unnecessary computation within a loop which can be easily refactored. The flame chart changes from:

Screenshot 2020-01-03 at 23 50 02

to:

Screenshot 2020-01-04 at 00 52 21

which is a small but not insignificant win 🎉 

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### Reviewers: @webdriverio/project-committers

### CODE REVIEW

**Comment by gustavohenke** [comment]
> which is a small but not insignificant win

I think it isn't a small one! The chart looks much cleaner now.

File: packages/webdriverio/src/utils/getElementObject.js [hunk 1]
```diff
@@ -61,10 +63,15 @@ export const getElement = function findElement (selector, res, isReactElement =
  * @param  {Object} res       findElement response
  * @return {Object}           WDIO element object
  */
-export const getElements = function getElements (selector, res, isReactElement = false) {
+export const getElements = function getElements (selector, elemResponse, isReactElement = false) {
     const browser = getBrowserObject(this)
+    const propertiesObject = {
```
Comment (erwinheitzman) [inline packages/webdriverio/src/utils/getElementObject.js]: I suggest that we keep the structure the same as in the previous method, so:
```
    const customCommandPrototype = clone(browser.__propertiesObject__)
    const elementPrototype = getWDIOPrototype('element')
    const propertiesObject = { ...customCommandPrototype, ...elementPrototype, scope: 'element' }
```

**Comment by christian-bromann** [comment]
@erwinheitzman I adjusted both implementations, anything left?

**Comment by erwinheitzman** [comment]
LGTM

**Pull request merged**